options/locale/locale_en-US.ini

```diff
@@ -1391,12 +1391,16 @@ issues.remove_labels = removed the %s labels %s
 issues.add_remove_labels = added %s and removed %s labels %s
 issues.add_milestone_at = `added this to the <b>%s</b> milestone %s`
 issues.add_project_at = `added this to the <b>%s</b> project %s`
+issues.add_project_board_at = `added this to <b>%s</b> %s`
 issues.change_milestone_at = `modified the milestone from <b>%s</b> to <b>%s</b> %s`
 issues.change_project_at = `modified the project from <b>%s</b> to <b>%s</b> %s`
+issues.change_project_board_at = `moved this to <b>%s</b> to <b>%s</b> %s`
```

Comment:
Shouldn't this be 'moved this **from** .. to ..?

Reply:
i apologize for my absent-mindedness.